app/forms/idv/api_document_verification_status_form.rb

```diff
@@ -31,12 +31,17 @@ def remaining_attempts
 
     def timeout_error
       return unless @async_state.timed_out?
-      errors.add(:timeout, t('errors.doc_auth.document_verification_timeout'))
+      errors.add(
+        :timeout, t('errors.doc_auth.document_verification_timeout'),
+        type: :doc_verification
+      )
     end
 
     def failed_result
       return if !@async_state.done? || @async_state.result[:success]
-      @async_state.result[:errors].each { |key, error| errors.add(key, error) }
+      @async_state.result[:errors].each { |key, error|
+ errors.add(key, error, type: :doc_verification)
+      }
```

Comment:
hopefully rubocop will catch this, but the indendation is wrong, and I wold also use `do...end` for multiline blocks

```suggestion
      @async_state.result[:errors].each do |key, error|
        errors.add(key, error, type: error)
      end
```

and the particular error message... we may want to set that as `error` here